Codeinwp/neve
# About Page updated

### Summary


Preview and Import buttons added in About page, when hovering over site item.

### Will affect visual aspect of the product

NO

### Screenshots 
![neve-onboarding](https://user-images.githubusercontent.com/26867648/47352222-26567580-d6c2-11e8-84d7-7f46e2bae40a.png)

### Test instructions


- Check the Sites Library tab in About page. Two buttons (preview and import) should appear when hovering over a site item.
